PhysicsTools/PatAlgos/plugins/PATMETSlimmer.cc

```diff
@@ -94,11 +94,13 @@ void pat::PATMETSlimmer::maybeReadShifts(const edm::ParameterSet &basePSet, cons
 	}
 	else if(level==pat::MET::Smear) {
 	  shifts_.push_back(OneMETShift(pat::MET::NoShift, level, baseTag, consumesCollector(), readFromMiniAOD, true, false, true));
+	  shifts_.push_back(OneMETShift(pat::MET::NoShift, level, baseTag, consumesCollector(), readFromMiniAOD, false, true, true));
```

Comment:
That line should be removed, as it will conflict with the storage of the MET type1, the way it is done, it will erase the type1MET to put the type1Smeared MET in place